album/@MUSHOKU-TENSEI-OST-1.toml

```
@@ -0,0 +1,101 @@
+[album]
+album_id = "ca5dc97b-4e3a-467c-957e-8765cb1d584b"
+title = "『無職転生～異世界行ったら本気出す～』 オリジナル・サウンドトラック1"
+artist = "藤澤慶昌"
+date = 2021-04-21
+type = "absolute"
+catalog = "@MUSHOKU-TENSEI-OST-1"
```

```suggestion
catalog = "TBR31094D-2"
```

This?
https://vgmdb.net/album/108263

---

但好像又对不上？

---

这个是 Special Edition